packages/toolkit/src/view/hub/hub-view/Body.tsx

```diff
@@ -139,14 +139,12 @@ export const Body = ({
             ) : (
               allPipelines.length &&
               allPipelines.map((pipeline) => (
-                <React.Fragment key={pipeline.id}>
-                  <CardPipeline
-                    ownerID={pipeline.owner_name.split("/")[1]}
-                    pipeline={pipeline}
-                    isOwner={pipeline.owner_name === me.data?.name}
-                    disabledPermissionLabel={true}
-                  />
-                </React.Fragment>
+                <CardPipeline
+                  ownerID={pipeline.owner_name.split("/")[1]}
+                  pipeline={pipeline}
+                  isOwner={pipeline.owner_name === me.data?.name}
+                  disabledPermissionLabel={true}
+                />
```

Comment:
@iamnamananand996 Even though this achieve desired behavior, I won't merge this PR into main because it violates the this important rendering tactics of React. 

Please take a deep look to understand why this happened